tests/functional/docs/test_help_output.py

```diff
@@ -432,3 +433,54 @@ def test_operation_help_command_has_note(self):
         self.assert_contains(
             'The default endpoint data.iot.[region].amazonaws.com is '
             'intended for testing purposes only.')
+
+
+class TestAliases(BaseAWSHelpOutputTest):
+    def setUp(self):
+        super(TestAliases, self).setUp()
+        self.files = FileCreator()
+        self.alias_file = self.files.create_file('alias', '[toplevel]\n')
+
+        # TODO: If an environment variable is ever exposed to
+        # modify the alias file location, remove this patch
+        # and use that environment variable because patching of specific
+        # classes properties should be avoided for functional tests but
+        # there is not really any other way to hook in...
+        self.alias_file_location_patch = mock.patch(
+            'awscli.alias.AliasLoader.ALIAS_FILENAME', self.alias_file)
+        self.alias_file_location_patch.start()
+
+    def tearDown(self):
+        super(TestAliases, self).tearDown()
+        self.files.remove_all()
+        self.alias_file_location_patch.stop()
+
+    def add_alias(self, alias_name, alias_value):
+        with open(self.alias_file, 'a+') as f:
+            f.write('%s = %s\n' % (alias_name, alias_value))
+
+    def test_alias_not_in_main_help(self):
+        self.add_alias('my-alias', 'ec2 describe-regions')
+        self.driver.main(['help'])
+        self.assert_not_contains('my-alias')
+
+    def test_alias_proxies_to_service_help(self):
```

Comment:
We might have chatted about this, but is this what we want here?  Seems like we'd maybe want to display the fact that this is an alias command.  I can see this being confusing just showing the help page for the underlying service command.  It also is odd that the `help` command will only work if you're proxying a service/operation with no args (otherwise you might get a parse error), so you can't tell if `aws myalias help` will work without knowing what the alias is.